tests/memoryview/memoryview_shared_utility.srctree

```diff
@@ -0,0 +1,79 @@
+# tag: numpy
+
+"""
+PYTHON -m Cython.Build.SharedModule pkg2
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import test_shared_utility"
+"""
+
+######## setup.py ########
+
+from Cython.Build import cythonize
+from setuptools import setup, Extension
+from Cython.Compiler import Options
+import numpy
+
+Options.use_shared_utility = 'pkg2'
+
+extensions = [
+    Extension("*", ["**/*.pyx"], include_dirs=[numpy.get_include()]),
+    Extension("pkg2.MemoryView", sources=["pkg2/MemoryView.c"])
+]
+
+setup(
+  ext_modules = cythonize(extensions),
+)
+
+######## test_shared_utility.py ########
```

Comment:
It would be good if the test also validated the import (-> `sys.modules`) and usage of the shared module. Maybe using `__module__` or `__qualname__` or so?